code/node-server/src/middleware/reqValidators/validID.js

```diff
@@ -2,7 +2,7 @@ const config = require('../../../config.js')
 
 function validID(req) {
   const id = req.query.id
-  return id < config.IDLIMIT['low'] || id > config.IDLIMIT['high'] ? false : true
+  return id <= config.IDLIMIT['low'] || id >= config.IDLIMIT['high'] ? true : false
```

Comment:
The logic for our `validID` was returning true even when `req.query.id` was undefined. It was also rejecting ids that were on the low or high end of our config. This should fix the issue